README.md

```diff
@@ -166,6 +167,58 @@ mysql::db { 'mydb':
 }
 ```
 
+### Install Percona server on Centos
+
+This example shows how to do a minimal installation of a Percona server on a Centos (has been tested on Puppet 4.4 / Centos 7 / Percona Server 5.7) including the Percona server, client and bindings.
+
+The configuration change here is really minimal.
+
+**Note:** The installation of the yum repository is not part of this package and is here only to show a full example of how you can install this.
+
+```puppet
+yumrepo { 'percona':
+  descr    => 'CentOS $releasever - Percona',
+  baseurl  => 'http://repo.percona.com/centos/$releasever/os/$basearch/',
+  gpgkey   => 'http://www.percona.com/downloads/percona-release/RPM-GPG-KEY-percona',
+  enabled  => 1,
+  gpgcheck => 1,
+}
+
+class {'mysql::server':
+  package_name     => 'Percona-Server-server-57',
+  service_name     => 'mysqld',
+  config_file      => '/etc/my.cnf',
+  includedir       => '/etc/my.cnf.d',
+  override_options => {
+    mysqld => {
+      log-error => '/var/log/mysqld.log',
+      pid-file  => '/var/run/mysqld/mysqld.pid',
+    },
+    mysqld_safe => {
+      log-error => '/var/log/mysqld.log',
+    },
+  }
+}
+
+class {'mysql::client':
+  package_name => 'Percona-Server-client-57',
+}
```

Comment:
It's not, it's just here to show how to install the client it you want to install it.